src/components/Scoremeter/index.tsx

```diff
@@ -49,20 +49,28 @@ const ScoreMeter = (props: ScoremeterProps) => {
     }, []);
 
     React.useEffect(() => {
+        let timerId: number;
         if (score !== sanitizedReading) {
-            setTimeout(() => {
+            timerId = window.setTimeout(() => {
                 setScoreUpdate(true);
             }, 500);
         }
+        return () => {
+            clearTimeout(timerId);
```

Comment:
If `score !== sanitizedReading` we might get the timerId? but if not are we doing a `clearTimeout(undefined)`

Reply:
Yes